conda.recipe/meta.yaml

```diff
@@ -1,27 +1,29 @@
-{% set data = load_setup_py_data() %}
+{% set __version__ = "0.6.0" %}
```

Comment:
OK, I understand now what you were talking about when saying that conda recipe stuff support less pyproject. Can we write some bash or python script in this field `{% set variable = script() %}` to fetch the version from `__init__.py` ?